src/services/quandl-service.js

```diff
@@ -12,6 +12,34 @@
         QUANDL_URL = 'https://www.quandl.com/api/v3/',
         QUANDL_WIKI = 'datasets/WIKI/';
 
+    var useApiKey = true;
+
+    var Observable = (function() {
+        var proto = Obs.prototype;
+        function Obs() {
+            this.registry = {};
+        }
+
+        proto.on = function(eventName, fn, scope) {
+            if (!this.registry[eventName]) {
+                this.registry[eventName] = [];
+            }
+
+            this.registry[eventName].push({
+                fn: fn,
+                scope: scope
+            });
+        };
+
+        proto.notify = function(eventName, value) {
+            this.registry[eventName].forEach(function(entry) {
+                entry.fn.call(entry.scope, value);
+            });
+        };
+
+        return Obs;
+    })();
```

Comment:
Can do, habit too over me there!
I made it like this so it's tech independent, meaning we won't have to touch it if we move to react.